demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -574,22 +573,41 @@ export class DemoMeetingApp
     });
 
     const buttonVideo = document.getElementById('button-camera');
+    const videoNotif = document.getElementById('no-video-notif');
+    let notifTimeout: number | null = null;
```

Comment:
This should be `undefined | ReturnType<typeof setTimeout>`.

Reply:
Hmm, Typescript tells me `setTimeout` returns a `number`. At least when calling `window.setTimeout` compared to just `setTimeout`, which I think the compiler thinks it might be running in a node environment that has a different return type or something?

Reply:
In a browser `setTimeout` returns a `number`. In Node it returns a structure. `ReturnType` means you don't have to worry about it (you cannot strictly type your TypeScript code for both environments without it!) and it's a nice expressive shorthand to say exactly what you mean, which is you'll use this slot to hold whatever it gives you.

Reply:
Sweet, haven't ran into `ReturnType` before but will definitely start using it